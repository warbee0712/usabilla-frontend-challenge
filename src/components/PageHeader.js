import React from 'react'
// Elements
import Box from '../elements/Box'
// Components
import Heading from './Heading'

function PageHeader() {
  return (
    <Box px="xl" py="m" position="relative" boxShadow="default">
      <Heading fontWeight={500}>
        <span role="img" aria-label="smiling face with heart-shaped eyes">
          😍
        </span>{' '}
        Dashboard
      </Heading>
    </Box>
  )
}

export default PageHeader
