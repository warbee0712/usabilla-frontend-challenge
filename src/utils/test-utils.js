import React from 'react'
import { render } from 'react-testing-library'
import { ThemeProvider } from 'styled-components'
import theme from '../theme'

const customRender = (node, options) => {
  return render(<ThemeProvider theme={theme}>{node}</ThemeProvider>, options)
}

// re-export everything
export * from 'react-testing-library'

// override render method
export { customRender as render }
