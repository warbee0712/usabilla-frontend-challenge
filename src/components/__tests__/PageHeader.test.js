import React from 'react'
import { render } from 'test-utils'
import PageHeader from '../PageHeader'

describe('PageHeader', () => {
  it('should match snapshot', () => {
    const { container } = render(<PageHeader />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
