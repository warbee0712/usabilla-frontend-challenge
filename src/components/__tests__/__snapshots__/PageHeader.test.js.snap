// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader should match snapshot 1`] = `
.c0 {
  box-shadow: 0 1px 2px rgba(0,0,0,0.15);
  position: relative;
  padding-left: 2.5rem;
  padding-right: 2.5rem;
  padding-top: 1rem;
  padding-bottom: 1rem;
}

.c1 {
  font-family: Milo,BlinkMacSystemFont,-apple-system,Segoe UI,Roboto,Helvetica,Arial,sans-serif;
  font-size: 1.1875rem;
  font-weight: 500;
  line-height: 1.6875rem;
}

<div
  class="c0"
>
  <h1
    class="c1"
    font-family="primary"
    font-size="s"
    font-weight="500"
  >
    <span
      aria-label="smiling face with heart-shaped eyes"
      role="img"
    >
      üòç
    </span>
     
    Dashboard
  </h1>
</div>
`;
